app/(afterLogin)/(timesheetInformation)/timesheet/settings/allowed-areas/_components/locationSidebar/index.tsx

```
@@ -184,104 +178,82 @@
 
-                        <Form.Item
-                          id="isGlobalLocation"
-                          label="Is Global"
-                          name="isGlobal"
-                        >
-                          <Switch
-                            defaultChecked
-                            onChange={(checked) => setShowUsers(!checked)}
-                          />
-                        </Form.Item>
+                  <div className="flex items-center gap-2 py-4">
```

ESLint

[error] Delete `··`

(prettier/prettier)